instrumentation/benchmarks/README.md

```diff
@@ -5,7 +5,13 @@ benchmarks for Brave instrumentation. You can use these to measure overhead
 of using Brave.
 
 ### Running the benchmark
-From the project directory, run `./mvnw install -pl instrumentation/benchmarks -am -DskipTests` to build the benchmarks, and the following to run them:
+From the project directory, run this to build the benchmarks:
+
+```bash
+$ ./mvnw install -pl instrumentation/benchmarks -am -Dmaven.test.skip.exec=true`
```

Comment:
ps the docs are the same as before except fixing this glitch. in the parent pom, the benchmarks module is removed when `-DskipTests`, so basically it was a broken doc.